api/golang/engine/lib/kurtosis_context/portal_daemon_client.go

```diff
@@ -32,7 +36,9 @@ func CreatePortalDaemonClient(mustBuildClient bool) (portal_api.KurtosisPortalCl
 		)
 	}
 	portalClient := portal_api.NewKurtosisPortalClientClient(portalConn)
-	_, portalReachableError := portalClient.Ping(context.Background(), portal_constructors.NewPortalPing())
+	ctxWithTimeout, cancelFunc := context.WithTimeout(context.Background(), waitForPortalClientPingTimeout)
```

Comment:
We give enough time for the connection to be ready to issue the Ping call.  This fixes the issue where the Ping call failed just after starting the Portal client.